README.md

```diff
@@ -14,7 +14,7 @@ This cookbook provides resources for working with [Habitat](https://habitat.sh).
 
 ### Platforms
 - RHEL 6+
-- Ubuntu 12.04+
+- Ubuntu 14.04+
```

Comment:
hab_install and hab_package will, hab_sup won't. hab_service would if hab-sup was set up some other way. I don't have cycles to implement a hab_sup_upstart, though.